bin/install-core

```diff
@@ -37,7 +37,11 @@ export DOTFILES=$SOURCE/dotfiles
 export LOCAL_DIR=$DOTFILES/.local
 
 DOWNLOAD_DIR=$HOME/.local/share/bitcoin-core
-
+BITCOIN_CORE_SOURCE_DOMAIN=(
+    "6hasakffvppilxgehrswmffqurlcjjjhd76jgvaqmsg6ul25s7t3rzyd.onion" #tor
```

Comment:
let's call that `onion service` or `Tor onion service`

Reply:
Fixed with 7895f60977905608f75a1fb3c408b03cc8e35bcc.